provider/oci/instance.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/juju/juju/environs/instances"
 	"github.com/juju/juju/instance"
 	"github.com/juju/juju/network"
+	providerCommon "github.com/juju/juju/provider/oci/common"
```

Comment:
No need for alias

Reply:
Yeah this was just for consistency, will fix.